gradle.properties

```diff
@@ -1,4 +1,4 @@
-version = 0.9.3-SNAPSHOT
+version = 0.9.4-SNAPSHOT
```

Comment:
don't change the version number at one PR

Reply:
ok，